docs/FAQ.rst

```diff
@@ -0,0 +1,276 @@
+Q&A
+***
+
+Q: How do I know if DALI can help me?
+#####################################
+A: You need to check our docs first and see if DALI operators cover your use case. Then try to run
+a couple of iterations of your training with a fixed data source - a random tensor that is generated
+once to see if you can train faster without any data processing. If so, then the data processing
+is a bottleneck in your case and DALI may help. This topic is covered in detail in
+`the GTC'22 talk <https://www.nvidia.com/gtc/session-catalog/#/session/1636559250287001p4DG>`_.
+
+Q: What data format does DALI support?
+######################################
+A: Regarding the data storage, DALI natively supports TFRecord, RecordIO, caffe/caffe2 LMDB,
+WebdataSet. Regarding the data types, it supports images, video, and audio. If the given type
+is not supported the external source operator can execute user-provided python code that would
+read and parse it.
```

Comment:
I think it mixes the two a bit.
```suggestion
A: DALI can load most of the image file formats (JPEG, PNG ,TIFF, JPEG2000 and more) as well as audio (WAV, OGG, FLAC) and video. The files can be stored individually or grouped in storage containers from common DL frameworks - TFRecord, RecordIO, caffe/caffe2 LMDB. WebDataset is also supported. In case there's no native support for a particular format, the data can be loaded and parsed in user-provided python code and supplied via external source.
```

Reply:
Fixed